src/main/webapp/UtoverDash/index.jsp

```diff
@@ -8,23 +8,31 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
     <link rel="stylesheet" href="../style.css">
     <link rel="stylesheet" href="../UtoverDash/style.css">
-</head>
+<head>
```

Comment:
Ser du fjernet </head>, og byttet ut med <head>. Dette blir feil. Det skal stå </head> for å avslutte headeren.

Reply:
< h e a d > skal byttes ut med < / h e a d >